protocol/x/clob/keeper/liquidations.go

```diff
@@ -222,34 +224,49 @@ func (k Keeper) PlacePerpetualLiquidation(
 
 	// TODO(DEC-1323): Potentially allow liquidating the same perpetual + subaccount
 	// multiple times in a block.
+	perpetualId := liquidationOrder.MustGetLiquidatedPerpetualId()
 	k.MustUpdateSubaccountPerpetualLiquidated(
 		ctx,
 		liquidationOrder.GetSubaccountId(),
-		liquidationOrder.MustGetLiquidatedPerpetualId(),
+		perpetualId,
 	)
 
-	telemetry.IncrCounter(
-		1,
-		metrics.Liquidations,
-		metrics.PlacePerpetualLiquidation,
-		metrics.Count,
-	)
+	labels := []gometrics.Label{
+		metrics.GetLabelForIntValue(metrics.PerpetualId, int(perpetualId)),
+		metrics.GetLabelForBoolValue(metrics.IsBuy, liquidationOrder.IsBuy()),
+	}
 
+	// Stat the number of liquidation orders placed.
 	telemetry.IncrCounterWithLabels(
-		[]string{metrics.Liquidations, metrics.PlacePerpetualLiquidation, metrics.BaseQuantums},
-		metrics.GetMetricValueFromBigInt(liquidationOrder.GetBaseQuantums().ToBigInt()),
-		[]gometrics.Label{
-			metrics.GetLabelForIntValue(metrics.PerpetualId, int(liquidationOrder.MustGetLiquidatedPerpetualId())),
-		},
+		[]string{metrics.Liquidations, metrics.PlacePerpetualLiquidation, metrics.Count},
+		1,
+		append(
+			[]gometrics.Label{
+				metrics.GetLabelForBoolValue(
+					metrics.Unfilled,
+					orderSizeOptimisticallyFilledFromMatchingQuantums == 0,
+				),
+				metrics.GetLabelForBoolValue(
+					metrics.FullyFilled,
+					orderSizeOptimisticallyFilledFromMatchingQuantums == liquidationOrder.GetBaseQuantums(),
+				),
+			},
+			labels...,
+		),
 	)
 
-	telemetry.IncrCounterWithLabels(
-		[]string{metrics.Liquidations, metrics.PlacePerpetualLiquidation, metrics.Filled, metrics.BaseQuantums},
-		metrics.GetMetricValueFromBigInt(orderSizeOptimisticallyFilledFromMatchingQuantums.ToBigInt()),
-		[]gometrics.Label{
-			metrics.GetLabelForIntValue(metrics.PerpetualId, int(liquidationOrder.MustGetLiquidatedPerpetualId())),
-		},
-	)
+	// Stat the volume of liquidation orders placed.
+	if totalQuoteQuantums, err := k.perpetualsKeeper.GetNetNotional(
+		ctx,
+		perpetualId,
+		liquidationOrder.GetBaseQuantums().ToBigInt(),
+	); err == nil {
+		telemetry.IncrCounterWithLabels(
+			[]string{metrics.Liquidations, metrics.PlacePerpetualLiquidation, metrics.QuoteQuantums},
+			metrics.GetMetricValueFromBigInt(totalQuoteQuantums),
+			labels,
+		)
```

Comment:
strongly consider some sort of distribution for these metrics

Then you can make analysis like "90% of liquidation orders are less than $10k in size". I think currently this only allows us to say "there were 30 liquidation events with a total amount of $100k" but not know if most of that is from a small number of orders, or if they are relatively similar size

Reply:
Will look into how `AddSample` from `go-metrics` works